tox.ini

```diff
@@ -42,8 +43,13 @@ commands =
 description = Run static analysis checks
 setenv =
     PYTHONPATH = ""
+    MYPYPATH={toxinidir}/lib
 commands =
-    mypy {[vars]all_path} {posargs}
+    mypy {[vars]src_path} {posargs}
+    mypy --python-version 3.9 {[vars]lib_path} {posargs}
```

Comment:
Is it really python 3.9 that ends up being used in the charm?

Reply:
Changed to `3.10`. Since Charmed 5G won't be used soon I think it's nice to assume it'll be used in jammy.